src/components/app/App.jsx

```diff
@@ -45,6 +46,9 @@ export default function App() {
               <PageRoute exact path="/:enterpriseSlug/program/:programUuid" component={ProgramPage} />
             )}
             <PageRoute exact path="/:enterpriseSlug/program-progress/:programUUID" component={ProgramProgressPage} />
+            {features.ENABLE_PROGRAM_PROGRESS_PAGE && (
+              <PageRoute exact path="/:enterpriseSlug/programs" component={ProgramListingPage} />
```

Comment:
I think `path="/:enterpriseSlug/programs"` will be `path="/:enterpriseSlug/programs-progress"` more suitable here.